### PR TITLE
fixes(#127)

### PR DESCRIPTION
**Pull request by MidSpike**

### CODE REVIEW

File: src/bot/commands/support.js [hunk 1]
```diff
@@ -126,7 +125,7 @@ const support_categories = new Discord.Collection([
     }, {
         id: 'TRANSFERS',
         name: 'Transfers',
-        description: 'Come here if you want to transfer any of your products to another account.',
+        description: 'Transfer or gift products to a new account.',
```
Comment (TotoTheEgg) [inline src/bot/commands/support.js:128]: Transfer or gift to a different account

File: src/bot/commands/support.js [hunk 2]
```diff
@@ -154,7 +153,7 @@ const support_categories = new Discord.Collection([
     }, {
         id: 'ISSUES',
         name: 'Issues',
-        description: 'Come here if you are having issues with our products.',
+        description: 'Tech support can be found here.',
```
Comment (TotoTheEgg) [inline src/bot/commands/support.js:156]: Product support can be found here

File: src/bot/commands/support.js [hunk 3]
```diff
@@ -434,139 +422,157 @@ module.exports = {
...
+                    embeds: [
+                        new Discord.MessageEmbed({
+                            color: 0x60A0FF,
+                            description: [
+                                `**Please read the [instructions](${category_instructions_message.url}) above!**`,
+                                '',
+                                `Send \"**done**\" after you have completed the [instructions](${category_instructions_message.url}).`,
+                                'Send \"**cancel**\" if you wish to cancel this ticket.',
+                                '',
+                                '*Sending neither will result in your ticket automatically closing after 30 minutes.*',
```
Comment (TotoTheEgg) [inline src/bot/commands/support.js:504]: Bot sends reminder message after 15 minutes if user has not said done?

**Review by TotoTheEgg — CHANGES_REQUESTED** [review]

File: src/bot/commands/support.js (hunk 1 above)
Comment (MidSpike) [inline src/bot/commands/support.js:128]:
```suggestion
        description: 'Transfer or gift to a different account',
```

File: src/bot/commands/support.js (hunk 2 above)
Comment (MidSpike) [inline src/bot/commands/support.js:156]:
```suggestion
        description: 'Technical support can be found here',
```

File: src/bot/commands/support.js (hunk 3 above)
Comment (MidSpike) [reply src/bot/commands/support.js:504]: comment taken under consideration but out of scope for PR

**Review by rossloika — APPROVED** [review]

**Review by WillFJS — APPROVED** [review]
Very happy to see the support command being changed to a drop-down menu instead of numbers

**Review by TotoTheEgg — APPROVED** [review]
Glad to see our members get an easier way to open tickets! As long as it is only the author can use the interaction it's perfect!

**Pull request merged**